templates/main.template.yaml

```diff
@@ -17,148 +17,146 @@ Metadata:
           - Section1eSumoLogicResourceRemoveOnDeleteStack
 
       - Label:
-          default: "CloudTrail app configuration"
+          default: "2.1 CloudTrail app configuration"
         Parameters:
-          - Section2InstallCloudTrailApp
-          - Section2InstallPCICloudTrailApp
-          - Section2InstallCISFoundationApp
-          - Section2InstallCloudTrailMonitoringAnalyticsApp
-          - Section2InstallCloudTrailSecOpsApp
+          - Section2aInstallCloudTrailApp
+          - Section2bInstallPCICloudTrailApp
+          - Section2cInstallCISFoundationApp
+          - Section2dInstallCloudTrailMonitoringAnalyticsApp
+          - Section2eInstallCloudTrailSecOpsApp
       - Label:
-          default: "S3 configuration"
+          default: "2.2 S3 configuration"
         Parameters:
-          - Section2CloudTrailCreateBucket
-          - Section2CloudTrailLogsBucketName
+          - Section2fCloudTrailCreateBucket
```

Comment:
There's 2 different scenarios of error happening related to this parameter:
1 - When selecting No for **Section2fCloudTrailCreateBucket** and using an AWS Cloudtrail trail managed by Control Tower we get a permissions error on SumoCloudTrailAppsStack CFn stack (/templates/apps/sumo-aws-apps/cloudtrail/cloudtrail.template.yaml) for SumoCloudTrailSource resource:
Received response status [FAILED] from custom resource. Message returned: ERROR: (truncated) S3 bucket 'bucketName=aws-controltower-logs-_MYAWSACCOUNTIDOBFUSCATED_-us-east-1' is not readable. The following permission is missing : 'missingPermission=ListBucketVersions'." } for url: https://api.us2.sumologic.com/api/v1/collectors/348688536/sources

2 - When selecting Yes for **Section2fCloudTrailCreateBucket** then it fails in a different step in SumoSecurityHubAppsStack Cfn stack (/templates/apps/sumo-aws-apps/security-hub/securityhub.template.yaml) in the SecurityHubConfiguration resource:
Received response status [FAILED] from custom resource. Message returned: ERROR: (truncated) rized to perform: iam:CreateServiceLinkedRole on resource: arn:aws:iam::_MYAWSACCOUNTIDOBFUSCATED_:role/aws-service-role/securityhub.amazonaws.com/AWSServiceRoleForSecurityHub because no identity-based policy allows the iam:CreateServiceLinkedRole action.

Reply:
@gcasilva this should be fixed, let us know if you are still getting same error